Tests/XCUITests/FxScreenGraph.swift

```diff
@@ -767,7 +769,7 @@ func createScreenGraph(for test: XCTestCase, with app: XCUIApplication) -> MMScr
         screenState.tap(app.buttons["newTabButtonTabTray"],
                         forAction: Action.OpenNewTabFromTabTray,
                         transitionTo: NewTabScreen)
-        screenState.tap(app.buttons["closeAllTabsButtonTabTray"], to: CloseTabMenu)
+        screenState.tap(app.toolbars.buttons["closeAllTabsButtonTabTray"], to: CloseTabMenu)
 //        screenState.tap(app.buttons["syncNowButtonTabsButtonTabTray"], to: CloseTabMenu)
```

Comment:
nit: do we need this commented line 🤔

Reply:
We are doing a refactor where we will update/remove unused lines on FxScreenGraph, like this one.